rasterview.desktop

```diff
@@ -1,11 +1,9 @@
 [Desktop Entry]
 Name=RasterView
 Comment=CUPS Raster Viewer
-TryExec=rasterview
 Exec=rasterview %F
-Icon=rasterview.png
+Icon=rasterview
 Terminal=false
 Type=Application
-MimeType=application/vnd.cups-raster
-Encoding=UTF-8
-Categories=Graphics;
+MimeType=image/x-cmu-raster;image/pwg-raster;image/urf;
```

Comment:
What is x-cmu-raster? The IANA registered MIME media type for CUPS raster is application/vnd.cups-raster.

Reply:
OK, can we add comments to the desktop file to indicate this?

(and unfortunate that this no-longer-used format gets priority... :/)

Reply:
sure, done :)